### PR TITLE
Add notebooks docs support

### PR DESCRIPTION
**Pull request by nastyachizhikova**
Add notebooks with new documentation for NER, SQuAD and classification. Add notebooks support to the config file. Minor fixed of references in other .rst files.

### CODE REVIEW

File: docs/integrations/socket_api.rst [hunk 1]
```diff
@@ -82,7 +82,7 @@ are used as model argument names instead of the list from model config's
 Socket client example (Python)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Socket client for :doc:`SQuAD </features/models/squad>` model with a batch of
+Socket client for :doc:`SQuAD </features/models/SQuAD>` model with a batch of
```
[the remark this answers is not in the document]
Comment (nastyachizhikova) [reply docs/integrations/socket_api.rst:85]: why would the path to documentation change?

File: docs/conf.py [hunk 2]
```diff
@@ -67,12 +69,19 @@
...
 # directories to ignore when looking for source files.
 # This pattern also affects html_static_path and html_extra_path .
-exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store', '**.ipynb_checkpoints']
+exclude_patterns = ['_build', 
+                    'Thumbs.db', 
+                    '.DS_Store', 
+                    '**.ipynb_checkpoints', 
+                    'features/models/classifiers.rst', 
+                    'features/models/ner.rst',
+                    'features/models/squad.rst'
```
[the remark this answers is not in the document]
Comment (nastyachizhikova) [reply docs/conf.py]: Fixed it in next commit as well as the next comments

**Review by IgnatovFedor — APPROVED** [review]

File: docs/features/models/NER.ipynb [hunk 3]
```diff
@@ -0,0 +1,671 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ygOmNC9R9l_y"
+   },
+   "source": [
+    "First make sure you have the DeepPavlov Library installed.\n",
+    "[More info about the first installation](http://docs.deeppavlov.ai/en/master/intro/installation.html)"
```
Comment (avsakharov) [inline docs/features/models/NER.ipynb:109]: "[More info about the first installation.](http://docs.deeppavlov.ai/en/master/intro/installation.html)"

File: docs/features/models/SQuAD.ipynb [hunk 4]
```diff
@@ -0,0 +1,679 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ygOmNC9R9l_y"
+   },
+   "source": [
+    "First make sure you have the DeepPavlov Library installed.\n",
+    "[More info about the first installation](http://docs.deeppavlov.ai/en/master/intro/installation.html)"
```
Comment (avsakharov) [inline docs/features/models/SQuAD.ipynb:155]: "[More info about the first installation.](http://docs.deeppavlov.ai/en/master/intro/installation.html)"

File: docs/features/models/NER.ipynb [hunk 5]
```diff
@@ -0,0 +1,671 @@
...
+   "outputs": [],
+   "source": [
+    "!python -m deeppavlov install ner_ontonotes_bert_torch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`ner_ontonotes_bert_torch` here is the name of the model's *config_file*. [What is a Config File?](http://docs.deeppavlov.ai/en/master/intro/configuration.html) \n",
```
Comment (avsakharov) [inline docs/features/models/NER.ipynb:143]: "`ner_ontonotes_bert_torch` is the name of the model's *config_file*. [What is a Config File?](http://docs.deeppavlov.ai/en/master/intro/configuration.html) \n",

File: docs/features/models/NER.ipynb [hunk 6]
```diff
@@ -0,0 +1,671 @@
...
+   },
+   "source": [
+    "## 3.2 Predict using CLI"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also get predictions in an interactive mode through CLI."
```
Comment (avsakharov) [inline docs/features/models/NER.ipynb:241]: "You can also get predictions in an interactive mode through CLI (Сommand Line Interface)."

File: docs/features/models/NER.ipynb [hunk 7]
```diff
@@ -0,0 +1,671 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There are two metrics that are used to evaluate a NER model in DeepPavlov:\n",
+    "\n",
+    "`ner_f1` is measured on the entity-level (actual text spans should match exactly)\n",
+    "\n",
+    "`ner_token_f1` is measured on a token level (correct tokens from not fully extracted entities will still be counted as TPs)"
```
Comment (avsakharov) [inline docs/features/models/NER.ipynb:291]: "`ner_token_f1` is measured on a token level (correct tokens from not fully extracted entities will still be counted as TPs (true positives))"

File: docs/features/models/NER.ipynb [hunk 8]
```diff
@@ -0,0 +1,671 @@
...
+    "To train the model on your data, you need to change the path to the training data in the *config_file*. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "5BC6SQW-9l_8"
+   },
+   "source": [
+    "Parse the *config_file* and change the path to your data from Python."
```
Comment (avsakharov) [inline docs/features/models/NER.ipynb:366]: Write on the previous line?

File: docs/features/models/NER.ipynb [hunk 9]
```diff
@@ -0,0 +1,671 @@
...
+     ]
+    }
+   ],
+   "source": [
+    "from deeppavlov import train_model\n",
+    "from deeppavlov.core.commands.utils import parse_config\n",
+    "\n",
+    "model_config = parse_config('ner_ontonotes_bert_torch')\n",
+    "\n",
+    "#  dataset that the model was trained on\n",
```
Comment (avsakharov) [inline docs/features/models/NER.ipynb:394]: "# dataset that the model was trained on\n",

File: docs/features/models/NER.ipynb [hunk 10]
```diff
@@ -0,0 +1,671 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {
+    "id": "QkqD3RJi9l_-"
+   },
+   "outputs": [],
+   "source": [
+    "# provide a path to the train file\n",
+    "model_config[\"dataset_reader\"][\"data_path\"] = \"contents/train.txt\""
```
Comment (avsakharov) [inline docs/features/models/NER.ipynb:427]: "model_config[\'dataset_reader\'][\'data_path\'] = \'contents/train.txt\'"

File: docs/features/models/NER.ipynb [hunk 11]
```diff
@@ -0,0 +1,671 @@
...
+    "```\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "NhubTsRJ9mAA"
+   },
+   "source": [
+    "The source text is **tokenized** and **tagged**. For each token, there is a tag with BIO markup. Tags are separated from tokens with **whitespaces**. Sentences are separated with **empty lines**."
```
Comment (avsakharov) [inline docs/features/models/NER.ipynb:485]: **BIO**?

File: docs/features/models/NER.ipynb [hunk 12]
```diff
@@ -0,0 +1,671 @@
...
+   "metadata": {},
+   "source": [
+    "# 6. Models list"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The table presents a list of all of the NER-models available in DeepPavlov Library."
```
Comment (avsakharov) [inline docs/features/models/NER.ipynb:578]: the DeepPavlov Library?

File: docs/features/models/NER.ipynb [hunk 13]
```diff
@@ -0,0 +1,671 @@
...
+    "| **PRODUCT**      | Vehicles, weapons, foods, etc. (Not services)          |\n",
+    "| **EVENT**        | Named hurricanes, battles, wars, sports events, etc.   |\n",
+    "| **WORK OF ART**  | Titles of books, songs, etc.                           |\n",
+    "| **LAW**          | Named documents made into laws                         |\n",
+    "| **LANGUAGE**     | Any named language                                     |\n",
+    "| **DATE**         | Absolute or relative dates or periods                  |\n",
+    "| **TIME**         | Times smaller than a day                               |\n",
+    "| **PERCENT**      | Percentage (including “%”)                             |\n",
+    "| **MONEY**        | Monetary values, including unit                        |\n",
+    "| **QUANTITY**     | Measurements, as of weight or distance                 |\n",
```
Comment (avsakharov) [inline docs/features/models/NER.ipynb:633]: Measurements such as weight or distance

File: docs/features/models/NER.ipynb [hunk 14]
```diff
@@ -0,0 +1,671 @@
...
+    "| **ORDINAL**      | “first”, “second”                                      |\n",
```
Comment (avsakharov) [inline docs/features/models/NER.ipynb:634]: “first”, “second”, etc.

File: docs/features/models/SQuAD.ipynb [hunk 15]
```diff
@@ -0,0 +1,679 @@
...
+    "!python -m deeppavlov install squad_bert"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "i6hPLgzvpBpn"
+   },
+   "source": [
+    "`squad_torch_bert` here is the name of the model's *config_file*. [What is a Config File?](http://docs.deeppavlov.ai/en/master/intro/configuration.html) \n",
```
Comment (avsakharov) [inline docs/features/models/SQuAD.ipynb:195]: `squad_bert` ?

`squad_bert` is the name of the model's *config_file*.

File: docs/features/models/SQuAD.ipynb [hunk 16]
```diff
@@ -0,0 +1,679 @@
...
+    "model = build_model('squad_bert', download=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ec572M3jpBpp"
+   },
+   "source": [
+    "**Input**: List[context, question]\n",
```
Comment (avsakharov) [inline docs/features/models/SQuAD.ipynb:248]: But in the further example two lists in Input: List[context], List[question]

File: docs/features/models/SQuAD.ipynb [hunk 17]
```diff
@@ -0,0 +1,679 @@
...
+    "\n",
+    "**Output**: List[answer, start_character, logit]"
```
Comment (avsakharov) [inline docs/features/models/SQuAD.ipynb:250]: What is the logit?

File: docs/features/models/SQuAD.ipynb [hunk 18]
```diff
@@ -0,0 +1,679 @@
...
+    "## 3.2 Predict using CLI"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "iE3j768spBps"
+   },
+   "source": [
+    "You can also get predictions in an interactive mode through CLI."
```
Comment (avsakharov) [inline docs/features/models/SQuAD.ipynb:294]: "You can also get predictions in an interactive mode through CLI (Command Line Interface)."

File: docs/features/models/SQuAD.ipynb [hunk 19]
```diff
@@ -0,0 +1,679 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "QkqD3RJi9l_-"
+   },
+   "outputs": [],
+   "source": [
+    "# provide a path to the train file\n",
+    "model_config[\"dataset_reader\"][\"data_path\"] = \"/contents/train-v1.1.json\""
```
Comment (avsakharov) [inline docs/features/models/SQuAD.ipynb:456]: "model_config[\'dataset_reader\'][\'data_path\'] = \'/contents/train-v1.1.json\'"

File: docs/features/models/SQuAD.ipynb [hunk 20]
```diff
@@ -0,0 +1,679 @@
...
+    "!python -m deeppavlov train squad_bert"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "AYXNJgap8HpL"
+   },
+   "source": [
+    "# 5. Evaluate "
```
Comment (avsakharov) [inline docs/features/models/SQuAD.ipynb:562]: F1 score and EM information in this section?

File: docs/features/models/SQuAD.ipynb [hunk 21]
```diff
@@ -0,0 +1,679 @@
...
+    "The table presents a list of all of the Context Question Answering models available in DeepPavlov Library."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "6QHx_g25pBpy"
+   },
+   "source": [
+    "| Config name  | Dataset | Language | Model Size | F1 score | EM  |\n",
```
Comment (avsakharov) [inline docs/features/models/SQuAD.ipynb:631]: best metrics bold?

**Pull request merged**